trainingRN/src/screens/todo/styles.js

```diff
@@ -3,16 +3,15 @@ import { StyleSheet} from 'react-native';
 export default StyleSheet.create({
```

Comment:
Place `export default` at the bottom of this file so we can easily put decorators / HOCs if we need them 😊